src/Cmfd.cpp

```diff
@@ -959,12 +962,12 @@ void Cmfd::updateMOCFlux() {
           iter != _cell_fsrs.at(i).end(); ++iter) {
 
           /* Set new flux in FSR */
-            _FSR_fluxes[*iter*_num_moc_groups+h] = getFluxRatio(i,h)
+          _FSR_fluxes[*iter*_num_moc_groups+h] = getUpdateRatio(i,h, *iter)
```

Comment:
inconsistent spacing between parameters ``i,h, *iter`` here and on the next line